### ISSUE TITLE
In the Wizard dialogue, switch users cannot select the Color, Power or Weakness options

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4618]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/wizards

[WCAG 2.1 Guideline]
2.1.1 Keyboard
(Level A)
https://www.w3.org/TR/WCAG21/#keyboard

[OS]
MacOS 10.15.3, Windows 10

[User Agents/Assistive Technologies]
Safari 13.0.5, Firefox 75.0, Switch

[Component]
Wizards	

[Description]
In the Wizard dialogue, switch users cannot select the Color, Power or Weakness options

[User Impact]
Switch users must simulate a mouse click in order to select these options. 	

[Steps to Reproduce]

See the following instructions for switch usage then follow the steps below.

Windows (see Scan through keys)
https://support.microsoft.com/en-us/help/10762/windows-use-on-screen-keyboard

Mac
https://support.apple.com/en-us/HT202865

1. Using a switch, navigate to the Wizard example and fill out the Basic Information.
2. Navigate to and activate the Next button.
3. Try navigating to and selecting a color.
For MacOS only: 
4. Navigate to and activate Pointer --&gt; Mouse Click. 
5. Select the vertical area of the click, and then the vertical coordinate of a color.
6. Select the horizontal area of the click, and then the horizontal coordinate of a color.&quot;	

[Recommendation]
The color block components must include markup that will provide both access to these controls via the keyboard &amp; ensure they contain accessible names and roles. A tabindex attribute with a value of &quot;0&quot; for one of the simulated radio buttons and a WAI-ARIA radio / radiogroup role should be implemented.	&quot;&lt;div _ngcontent-clarity-c220=&quot;&quot;&quot;&quot; class=&quot;&quot;clr-row wizdemo-color-block-row&quot;&quot; role=&quot;&quot;radiogroup&quot;&quot; aria-labelledby=&quot;&quot;clr-id-4&quot;&quot; aria-describedby=&quot;&quot;[id for color selection content]&quot;&quot;&gt;
        &lt;div _ngcontent-clarity-c220=&quot;&quot;&quot;&quot; class=&quot;&quot;wizdemo-color-block-wrapper clr-col-3 ng-star-inserted&quot;&quot; style=&quot;&quot;&quot;&quot; role=&quot;&quot;radio&quot;&quot; aria-label=&quot;&quot;Blue&quot;&quot; aria-checked=&quot;&quot;true&quot;&quot; tabindex=&quot;&quot;0&quot;&quot;&gt;
                &lt;div _ngcontent-clarity-c220=&quot;&quot;&quot;&quot; class=&quot;&quot;wizdemo-color-block active&quot;&quot; style=&quot;&quot;background-color: blue;&quot;&quot;&gt; &amp;nbsp; &lt;/div&gt;
        &lt;/div&gt;
        ...
&lt;/div&gt;&quot;

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
